lib/IRGen/GenDistributed.cpp

```diff
@@ -514,6 +580,35 @@ void DistributedAccessor::decodeArgument(unsigned argumentIdx,
   }
 }
 
+static llvm::Value *lookupWitnessTable(IRGenFunction &IGF, llvm::Value *witness,
+                                       ProtocolDecl *protocol) {
+  assert(Lowering::TypeConverter::protocolRequiresWitnessTable(protocol));
+
+  auto &IGM = IGF.IGM;
+  auto *protocolDescriptor = IGM.getAddrOfProtocolDescriptor(protocol);
+  auto *witnessTable = IGF.Builder.CreateCall(
+      IGM.getConformsToProtocolFunctionPointer(), {witness, protocolDescriptor});
+
+  auto failBB = IGF.createBasicBlock("missing-witness");
+  auto contBB = IGF.createBasicBlock("");
+
+  auto isNull = IGF.Builder.CreateICmpEQ(
+    witnessTable, llvm::ConstantPointerNull::get(IGM.WitnessTablePtrTy));
+  IGF.Builder.CreateCondBr(isNull, failBB, contBB);
+
+  // This operation shouldn't fail because the compuler should have
```

Comment:
```suggestion
  // This operation shouldn't fail because the compiler should have
```

not worth a test re-run though ;)

Reply:
Fixed, thank you!